File: mycuration/src/main/java/com/phicdy/mycuration/view/CurationListView.java
package com.phicdy.mycuration.view;

import com.phicdy.mycuration.rss.Curation;

import java.util.ArrayList;

public interface CurationListView {
    void startEditCurationActivity(int editCurationId);
    void setNoRssTextToEmptyView();
    void setEmptyViewToList();
    void registerContextMenu();
    void initListBy(ArrayList<Curation> curations);
    void delete(Curation curation);
    int size();
    Curation curationAt(int position);
}

File: feature_curation_list/src/main/java/com/phicdy/mycuration/curationlist/CurationListView.kt
package com.phicdy.mycuration.curationlist

import com.phicdy.mycuration.entity.Curation

interface CurationListView {
    fun startEditCurationActivity(editCurationId: Int)
    fun setNoRssTextToEmptyView()
    fun registerContextMenu()
    fun initListBy(curations: List<Curation>)
    fun showRecyclerView()
    fun hideRecyclerView()
    fun showEmptyView()
    fun hideEmptyView()
}
